tests/keras/layers/test_convolutional.py

```diff
@@ -369,8 +369,14 @@ def test_zero_padding_2d():
     stack_size = 2
     input_nb_row = 11
     input_nb_col = 12
+    dim_ordering = K.image_dim_ordering()
+    print("Dim ordering: {}".format(dim_ordering))
```

Comment:
Inconsistent use of `"` (should be `'`). Also not clear that this statement is necessary.